pkg/commands/isrunning/isrunning.go

```diff
@@ -0,0 +1,85 @@
+package isrunning
```

Comment:
I know the name is bad, but naming for me is hard so I just did what I though was right

Reply:
chgm?

Reply:
Let's do that :)